### ISSUE TITLE
Error:[caliper] [caliper-local-client.js] 	Initialization ongoing... runs in an infinite loop for the caliper benchmark test for Fabcar in HYperledger Fabric Samples

### ISSUE THREAD

**Issue by swathidhamo** [hyperledger/caliper#788]
**Brief descritption**
 I am trying to run the benchmark tests Hyperledger fabric fabcar project with a couch db network. 
I had trouble initially due to a process monitor issue so I had made the mentioned changes suggested by @tzortzispanagiotis  on the config.yaml file from the following issue. 
 https://github.com/hyperledger/caliper/issues/781
_They now lead to an error because the "Initialization ongoing" log from the caliper-local-client.js file runs in an infinite loop_. This seems to be a core caliper engine so I don't know how to diagnose it. 

Looking at the file I realize the error comes from line 345/346 of the caliper-local-client.js file, it seems to be because the context is not set. 
I have attached the config and fabric go files with a couple of other syntax changes. 

Command : ` npx caliper launch master  --caliper-workspace .  --caliper-benchconfig benchmarks/samples/fabric/fabcar/config.yaml --caliper-networkconfig networks/fabric/fabric-v1.4.1/2org1peercouchdb/fabric-go.yaml
`
Version used: 0.3.0
Environment name and version (e.g. Chrome 39, node.js 5.4): Nodejs v10.19, npm v5.6.0
Operating System and version (desktop or mobile): Ubuntu 18.04

**This is the snapshot of the caliper-local-client file**
![Screenshot from 2020-04-04 18-31-42](https://user-images.githubusercontent.com/22546722/78451331-a7c97a00-76a2-11ea-80c6-5ce1a782992d.png)

**The command line logs**
![Screenshot from 2020-04-04 18-33-30](https://user-images.githubusercontent.com/22546722/78451365-dc3d3600-76a2-11ea-941c-2f8ee7ccf656.png)
![Screenshot from 2020-04-04 18-33-34](https://user-images.githubusercontent.com/22546722/78451370-df382680-76a2-11ea-857f-f0fc9c7c21c2.png)
![Screenshot from 2020-04-04 18-33-39](https://user-images.githubusercontent.com/22546722/78451372-e0695380-76a2-11ea-9df0-cfbdcba6dba0.png)
![Screenshot from 2020-04-04 18-33-43](https://user-images.githubusercontent.com/22546722/78451373-e19a8080-76a2-11ea-8304-e56c314b7ec7.png)

**Help here would be much appreciated! I have tried to modify that file and trace errors but I am unable to do so. And a switch to version 0.2.0 also yields a similar error.**

**The fabric-go.yaml file**
[fabric-go.txt](https://github.com/hyperledger/caliper/files/4431383/fabric-go.txt)

**The config.yaml file**
[config.txt](https://github.com/hyperledger/caliper/files/4431384/config.txt)

_Originally posted by @swathidhamo in https://github.com/hyperledger/caliper/issues/781#issuecomment-609025809_

Thank you in advance

**Comment by nklincoln** [comment]
any information from within the peer or chaincode containers?

**Comment by swathidhamo** [comment]
I was able to run the fabcar tests successfully with version 0.3.0. 
Turns out all it took was some modification of the assets field and then a few basic changes. 
I have attached the final config files for anyone's reference. 
Thank you @nklincoln for your help, taking a peek at the containers was what helped




[fabric-go.txt](https://github.com/hyperledger/caliper/files/4451491/fabric-go.txt)
[config.txt](https://github.com/hyperledger/caliper/files/4451489/config.txt)
